### ISSUE TITLE
Stanza editing tool API testing 

### ISSUE THREAD

**Issue by laurensorensen** [sul-dlss/speech-to-text#52]
[Stanza](https://www.telestream.com/captions/) is a propriety application that can be run from a server or desktop. One of the outcomes of our planning activity was investigating the API and if it functions the way we want it to. Further testing criteria may come during meeting Friday Dec 6, but this spreadsheet (header rows) is a start at what we'd like to discover: https://docs.google.com/spreadsheets/d/1l19rK_iAsBWkUFXLoZIstFseYnO_rTJMZ1wMnsLslhc/edit?gid=397321299#gid=397321299

Edit to add: Team will need to request a VM from Ops.

**Comment by laurensorensen** [comment]
Put in backlog until eval license software comes through.

**Comment by edsu** [comment]
I'm noticing that the Stanza downloads are for Windows and MacOS only. I think we may be able to request a WindowsVM but it might be easier to just test the API with the software running on our workstation.

**Comment by edsu** [comment]
Agreed, it is a mark against Stanza I think. I inquired in Slack to see if getting a MacOS or Windows VM is even a possibility.

**Comment by laurensorensen** [comment]
Ok, they confirmed we can use the old download links. Going to write back and confirm the end date of the trial. Copied below with the license key. Any word from Ops about using a VM? 

Download the Stanza installer for Windows Online: [Link to the Windows Stanza Installer](https://urldefense.com/v3/__https://www.telestream.net/download-files/stanza/Stanza-Windows.zip__;!!G92We9drHetJ8EofZw!ZbYFLA0nc2q996uEoXqvv0Axc6gw1ZlzAkdW0iAVuc-g_2BZHvih_lx1JHe66ZHq59JYab9KLElXViNCxutjbQPz$) 
Download the Stanza installer for macOS Online: [Link to the macOS Stanza Installer](https://urldefense.com/v3/__https://www.telestream.net/download-files/stanza/Stanza-Mac-Arm.zip__;!!G92We9drHetJ8EofZw!ZbYFLA0nc2q996uEoXqvv0Axc6gw1ZlzAkdW0iAVuc-g_2BZHvih_lx1JHe66ZHq59JYab9KLElXViNCxogxHbtB$)

Stanza Release Notes Online:  [Link to Stanza Release Notes](https://urldefense.com/v3/__https://www.telestream.net/download-files/stanza/Stanza-Release-Notes.pdf__;!!G92We9drHetJ8EofZw!ZbYFLA0nc2q996uEoXqvv0Axc6gw1ZlzAkdW0iAVuc-g_2BZHvih_lx1JHe66ZHq59JYab9KLElXViNCxtViQ2pJ$) 

Stanza User Guide Online: [Link to Stanza User Guide](https://urldefense.com/v3/__https://www.telestream.net/pdfs/user-guides/Stanza-User-Guide.pdf__;!!G92We9drHetJ8EofZw!ZbYFLA0nc2q996uEoXqvv0Axc6gw1ZlzAkdW0iAVuc-g_2BZHvih_lx1JHe66ZHq59JYab9KLElXViNCxjkInc13$)  

This is your trial license key for Stanza, valid for 30 days. Please keep this key for future use: 

License Key:   
SZ4A-YEZW-FFGB-W3SL-3D8F

**Comment by laurensorensen** [comment]
Here is the ops ticket for Windows VM: https://github.com/sul-dlss/operations-tasks/issues/3931

**Comment by edsu** [comment]
VM is ready for installing Stanza.

**Comment by laurensorensen** [comment]
Had an issue with installing to D: drive, asked for help here: https://github.com/sul-dlss/operations-tasks/issues/3931#issuecomment-2532871489

**Comment by laurensorensen** [comment]
Ok, I think I installed it correctly. I'm not sure how we want users to access Stanza. This is their user guide which says that you navigate to a browser: http://localhost:7000 in order to have it run, which I've done. But then, I am not sure how that works when there are multiple people trying to access?

**Comment by laurensorensen** [comment]
@dnoneill Asks if there is a way to port forward?

**Comment by laurensorensen** [comment]
@dnoneill says: "We should be able to upload files and create a new project. This means the button to open in stanza would work. After a project has been created we will have to do some really terrible stuff with the API to get the correct ID in stanza.
We should be able to get captions out of stanza but is going to be in my opinion horrible. We will have to either set up a watcher to watch a project status or watch for when a file is exported. Both aren't great. In order to get it out we will have to go on the shared mount and grab the file."
